docs/guide/generate_crud.md

```diff
@@ -0,0 +1,107 @@
+# Vaahcms
+
+[comment]: <> ([[toc]])
+
+## Introduction
+
+This is a guide to generate crud in VaahCms.
+Before jumping to crud part project should have generated module and activated.
+For the module and theme set up follow below link:-
+[https://github.com/webreinvent/vaahcli](https://github.com/webreinvent/vaahcli)
+
+## Step-1
+Firstly need to update VaahCms self update
+Use below command for that :
+```
+npm install vaah -g
+```
+## Step-2
+Use the below command to generate CRUD
+Use below command for that :
+```
+vaah cms:crud or npx vaah cms:crud
+```
+## Step-3
+Use following procedures to set up CRUD
+
+- For which you want to create CRUD: (Choose desired template using upper and lower arrow key then press enter)
+
+- Enter the Module/Theme/Entity name: (Enter generated module name)
+
+- Enter the section name (Backend | Frontend or Folder name): (For module choose backend, for theme choose frontend)
+
+- Vue folder name/path:(vue)
+
+- Enter your database table name:(Enter the table name for which you are genrating crud)
+
+- Do you want to generate migration file (true/false):  (true)
```

Comment:
@wri-abinash-s001 @we-vikram-wri231 explain each point, instead of write the value only

Reply:
@themodernpk Ok Sir,
I already tell him to explain each point, and you can use image or video for explain if needed

Reply:
@we-vikram-wri231 try to write comments on github

Reply:
Ok @themodernpk Sir,
I will do this

Reply:
@themodernpk noted sir